pkgs/servers/sql/postgresql/ext/pgroonga.nix

```diff
@@ -10,7 +10,17 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ pkg-config ];
-  buildInputs = [ postgresql msgpack-c groonga ];
+  buildInputs =
+    let
+      groonga-with-msgpack = groonga.overrideAttrs (oa: {
```

Comment:
Or should it be `groonga.override { suggestSupport = true; }`?

Reply:
`pgroonga` doesn't seem to use the suggest feature from `groonga`, would you recommend that we still use `suggestSupport` flag to build `groonga`?

Reply:
According to the upstream documentation: https://groonga.org/docs/install/debian.html#build-from-source, msgpack should be added as a dependency by default. So another choice would be to include `msgpack` (or `msgpack-c` if supported) in `buildInputs` unconditionally.

Reply:
I second this, It also a default dependency according to groonga's [brew formula](https://formulae.brew.sh/formula/groonga).

Reply:
Have added msgpack-c as default dependency, thanks for the review.